ArduCopter/Parameters.cpp

```diff
@@ -1111,6 +1111,46 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("SURFTRAK_MODE", 51, ParametersG2, surftrak_mode, (uint8_t)Copter::SurfaceTracking::Surface::GROUND),
 
+    // @Param: PILOT_RATE_RP_TC
+    // @DisplayName: Pilot Roll/Pitch Rate control input time constant
+    // @Description: Pilot roll and pitch rate control input time constant.  Low numbers lead to sharper response, higher numbers to softer response
+    // @Units: s
+    // @Range: 0 1
+    // @Increment: 0.01
+    // @Values: 0.5:Very Soft, 0.2:Soft, 0.15:Medium, 0.1:Crisp, 0.05:Very Crisp
+    // @User: Standard
+    AP_GROUPINFO("PILOT_RATE_RP_TC", 52, ParametersG2, pilot_rate_rp_tc, 0.05f),
```

Comment:
@bnsgeyer, I'm not sure either but I'm planning on removing Sport mode so I wouldn't worry about that mode.

Reply:
Yes, we don't need this and Sport can use the acro time constants.

Reply:
> @rmackay9 need your help with how to pass in the time constants. I took my best guess at doing this. Please review and let me know what you think. It seems like a lot to pass them in on every cycle. I thought that we might be able to get away with passing them in when you enter the flight mode through the INIT function. However if someone was to make a change to a parameter while in the flight mode it would never be seen by the attitude controller. Looking for guidance.

I think what you have done is fine for now.

Reply:
removed this param.  Thanks.